Assets/js/source-revenue.js

```diff
@@ -149,6 +156,13 @@ function renderCampaignName (row) {
     return '<a href="./campaigns/view/'+ row[1] +'" class="campaign-name-link" title="'+ row[2] + '">'+ row[2] + '</a>';
 }
 
+function renderMarginPercentage (row) {
```

Comment:
This function and the one below should be made callbacks or moved under the parent function. 

In general we should reduce the window.xxx global functions we declare as much as possible.